web/src/components/modals/modals.module.scss

```diff
@@ -1,12 +0,0 @@
-.greenButton {
-  --buttonBg: var(--app-accent-green);
-  --buttonTextColour: #fff;
-}
-
-.modalBody {
-  .errorText {
-    max-width: 300px;
-    font-weight: bold;
-    color: var(--app-error-colour);
-  }
-}
```

Comment:
Was this file meant to be deleted? It seems to be still being used by `shared/studio/components/modals/createDatabase.tsx`

Reply:
hey @jaclarke there is completely same file modals.module.scss inside shared/studio/components/modals. I believe this one is used, and the one I deleted is not used anywhere.

Reply:
Oops. Yes, you're right 🤦‍♂️.